### PR TITLE
decomp: `logic-target`

### LINKED ISSUE #1859: [decompiler] special case `(the bitfield-enum -1)` with all 64 bits set
Instead of some terrible `(bitfield-enum bit0 bit1 ... bit63)`

### PR DESCRIPTION
**Pull request by Hat-Kid**
Also adds all names for `focus-status`.

Closes #1859 and updates ref tests for Jak 1 for this edge case.

### CODE REVIEW

File: goal_src/jak2/engine/collide/collide-shape-h.gc [hunk 1]
```diff
@@ -78,6 +78,43 @@
...
+  (unknown-54 54)
+  (unknown-55 55)
+  (unknown-56 56)
+  (unknown-57 57)
+  (unknown-58 58)
+  (unknown-59 59)
+  (unknown-60 60)
+  (unknown-61 61)
+  (unknown-62 62)
+  (unknown-63 63)
```
Comment (ManDude) [inline goal_src/jak2/engine/collide/collide-shape-h.gc:117]: don't need this anymore I think

**Review by water111 — APPROVED** [review]
looks good!
I think there's some cleanup that we can eventually do (both decompiler bug fixes/improvements, and type stuff), but we can't really do it yet.
